packages/dashboard/src/instance/instanceDetails.tsx

```diff
@@ -3,6 +3,21 @@ import ChevronRightIcon from '@mui/icons-material/ChevronRight';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 import TreeItem from '@mui/lab/TreeItem';
 import TreeView from '@mui/lab/TreeView';
+import aws from 'aws-sdk';
+import { AssetDownloadInstruction } from '@sorry-cypress/common';
+import {
+  S3_ACL,
+  S3_BUCKET,
+  S3_REGION,
+  S3_PRESIGN_DOWNLOAD_URLS,
+  S3_FORCE_PATH_STYLE,
+  S3_READ_URL_PREFIX,
+  S3_IMAGE_KEY_PREFIX,
+  S3_VIDEO_KEY_PREFIX,
+  S3_DOWNLOAD_EXPIRY_SECONDS,
+} from '../../../director/src/screenshots/s3/config';
```

Comment:
I am not sure you can use the config values (in runtime) from other package. One approach would be to generate signed URLs on the API side